osquery/database/tablecachedb.cpp

```diff
@@ -0,0 +1,94 @@
+#include <osquery/database.h>
```

Comment:
This will need the standard license comment header.

Reply:
header in place